puppet-checks/src/main/java/com/iadams/sonarqube/puppet/checks/SingleQuotedStringContainingVariablesCheck.java

```diff
@@ -51,7 +51,7 @@ public void init() {
 
   @Override
   public void visitNode(AstNode node) {
-    if (CheckStringUtils.containsVariable(node.getTokenValue())) {
+    if (CheckStringUtils.containsVariable(node.getTokenValue()) && !("defined".equals(node.getTokenValue()))) {
```

Comment:
I'm not actually sure how to do this...

I basically want to say "if given node == `defined` then ignore as it's valid`, 

@iwarapter can you help me? :smile: